app/controllers/admin/manage_users/deactivated_users_controller.rb

```diff
@@ -1,22 +1,32 @@
 module Admin
   module ManageUsers
     class DeactivatedUsersController < ManageUsersController
+      def index
+        @users = User.deactivated.page(params[:page])
+      end
+
       def new
-        @user = user_scope.find(params[:active_user_id])
+        @user = User.active.find(params[:id])
       end
 
       def create
-        user = user_scope.find(params[:active_user_id])
+        user = User.active.find(params[:id])
         user.deactivate!
 
         set_flash :user_deactivated, user_name: user.name
         redirect_to admin_manage_users_root_path
       end
 
-      private
+      def confirm_reactivate
+        @user = User.find(params[:id])
```

Comment:
Should this not be within the 'active' users scpe as per line 13?

Reply:
good spot.